package.json

```diff
@@ -122,16 +121,12 @@
   "devDependencies": {
     "@faker-js/faker": "^7.6.0",
     "@axe-core/puppeteer": "^4.1.1",
-    "@babel/cli": "^7.10.5",
-    "@babel/core": "^7.11.4",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-transform-async-to-generator": "^7.10.4",
-    "@babel/plugin-transform-runtime": "^7.11.0",
```

Comment:
It's not strictly necessary to use `@babel/plugin-transform-runtime` if we are already using `core-js` with `useBuiltIns: 'usage'` in `@babel/preset-env`